.openpublishing.redirection.json

```diff
@@ -18,6 +18,14 @@
         "redirect_url": "/dotnet/articles/core/tools/dotnet-test"
     },
     {
+        "source_path": "docs/csharp/csharp-6.md", 
+        "redirect_url": "/dotnet/articles/csharp/whats-new/csharp-6"
```

Comment:
let's add the  "redirect_document_id": true to these entries as well, except for the docs/csharp/getting-started/whats-new.md.